ci/prow-ocm-adoption.sh

```diff
@@ -1,77 +1,77 @@
 #!/bin/bash
-
 set -o pipefail
 
-mkdir -p installer
-
 export VERSION=latest
 export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}')
+export OCM_URL="https://api.stage.openshift.com"
 
-export cmd=openshift-install
-export pullsecret_file=~/Downloads/crc-pull-secret.txt
-export extract_dir=$(pwd)
-
-if [ -z "$OCM_TOKEN" ]; then
-    echo "Assuming the token should be read from Prow";
+if [ -z "${OCM_TOKEN+x}" ]; then
+    echo "Assuming the OCM token should be read from Prow";
     export OCM_TOKEN=$(cat /usr/local/osde2e-credentials/ocm-refresh-token)
 fi
 
-if [ -z "AWS_ACCESS_KEY_ID" ]; then
+if [ -z "${AWS_ACCESS_KEY_ID+x}" ]; then
+    echo "Assuming the AWS Access token should be read from Prow";
     export AWS_ACCESS_KEY_ID=$(cat /usr/local/osde2e-credentials/aws-access-key-id)
 fi
 
-if [ -z "AWS_SECRET_ACCESS_KEY" ]; then
+if [ -z "${AWS_SECRET_ACCESS_KEY+x}" ]; then
+    echo "Assuming the AWS Secret token should be read from Prow";
     export AWS_SECRET_ACCESS_KEY=$(cat /usr/local/osde2e-credentials/aws-secret-access-key)
 fi
 
+if [ -z "${PULL_SECRET_FILE+x}" ]; then
+    echo "Assuming the Pull Secret should be read from Prow";
+    export PULL_SECRET_FILE=/usr/local/osde2e-credentials/stage-ocm-pull-secret
+fi
 
-cp /usr/local/osde2e-credentials/stage-installer-config ./installer/installer-config.yaml
+if [ -z "${INSTALLER_CONFIG+x}" ]; then
+    echo "Assuming the Installer Config should be read from Prow";
+    export INSTALLER_CONFIG=/usr/local/osde2e-credentials/stage-installer-config
+fi
 
+echo ${SHARED_DIR}
+echo "----";
+ls -lah ${SHARED_DIR}
+echo "----";
 
+export OCM_CONFIG=./.ocm.json
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
 
 curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/openshift-client-linux.tar.gz | tar zxvf - oc
 
-./oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
-
-./openshift-install create cluster --dir=$(pwd)/installer/ --log-level info
-
-export OCM_CONFIG=$(pwd)/.ocm.json
-export KUBECONFIG=$(pwd)/installer/auth/kubeconfig
+chmod +x oc
 
-{
-
-go get -u github.com/openshift-online/ocm-cli/cmd/ocm
+go get -u github.com/openshift-online/ocm-cli/cmd/ocm@master
```

Comment:
```suggestion
GO111MODULE=on go get github.com/openshift-online/ocm-cli/cmd/ocm@master
```

The `-u` flag (especially when used in non-module mode) is dangerous. It can install incorrect dependency versions, as it means "upgrade all dependencies of the target package to their highest possible version".

This formulation of the command should reliably install the expected code.